doc/release-notes/6.0-release-notes.md

```diff
@@ -0,0 +1,296 @@
+# Dataverse 6.0
+
+This is a platform upgrade release. Payara, Solr, and Java have been upgraded. No features have been added to the Dataverse software itself. Only a handful of bugs were fixed.
+
+Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project!
+
+## Release Highlights (Major Upgrades, Breaking Changes)
+
+This release contains major upgrades to core components. Detailed upgrade instructions can be found below.
+
+### Runtime
+
+- The required Java version has been increased from version 11 to 17.
+    - See PR #9764 for details.
+- Payara application server has been upgraded to version 6.2023.8.
+    - This is a required update.
+    - Please note that Payara Community 5 has reached [end of life](https://www.payara.fish/products/payara-platform-product-lifecycle/)
+    - See PR #9685 and PR #9795 for details.
+- Solr has been upgraded to version 9.3.0.
+    - See PR #9787 for details.
+
+### Development
+
+- Removal of Vagrant and Docker All In One (docker-aio), deprecated in Dataverse v5.14. See PR #9838 and PR #9685 for details.
+- All tests have been migrated to use JUnit 5 exclusively from now on. See PR #9796 for details.
+
+## Installation
+
+If this is a new installation, please follow our [Installation Guide](https://guides.dataverse.org/en/latest/installation/). Please don't be shy about [asking for help](https://guides.dataverse.org/en/latest/installation/intro.html#getting-help) if you need it!
+
+If you would like to be on our [map of Dataverse installations](https://dataverse.org/installations) around the world, please [create an issue](https://github.com/IQSS/dataverse-installations/issues) or email us at support@dataverse.org!
```

Comment:
I think this is fine, but I also wonder if we can be a little more direct - instead of "if you'd like", something more like "Once you're using Dataverse in a production setting, please let us know so we can add you to the map..."

Reply:
Thanks, I wrote that quickly with the main goal of removing the version number (5.14, 6.0, etc.)

I just reworded it a bit in d481c611ecc631e5dff2706fb9027433a33c6097

I almost threw a line in there about joining the GDCC. 🤔 I could definitely go add it to https://github.com/IQSS/dataverse-installations/blob/main/.github/ISSUE_TEMPLATE/add-dataverse-repository-to-dataverse-map.md

Reply:
I like the idea of also mentioning GDCC. 👍

Reply:
Ok, I added that too: 7259b35